test/integration/custom-server/server.js

```diff
@@ -25,7 +25,7 @@ app.prepare().then(() => {
     }
 
     if (/test-index-hmr/.test(req.url)) {
-      return app.render(req, res, '/index')
+      return app.render(req, res, '/')
```

Comment:
I believe this is a breaking change, and we needed to keep it for backwards compat.